src/components/dashboard/SalesByDayOfWeekChart.tsx

```
@@ -0,0 +1,171 @@
+  const { theme } = useTheme()
+  const isDark = theme === 'dark'
+
+  // Ensure we have all 7 days in the correct order
+  const dayOrder = ['Dom', 'Seg', 'Ter', 'Qua', 'Qui', 'Sex', 'Sab']
+  const chartData = dayOrder.map(day => {
+    const found = data.find(d => d.day_of_week === day)
+    return found || { day_of_week: day, sales: 0, percentage: 0 }
+  })
+
+  // Custom label component for percentages
+  const renderCustomLabel = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)